swap/swap.go

```diff
@@ -30,12 +30,14 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	l "github.com/ethereum/go-ethereum/log"
```

Comment:
I am not sure I understand why we import the same package twice but with different reference?

Reply:
Thanks for noticing that mistake.
I was importing the swarm log with l.
Later I changed the methods and I think it was reimported using the same package as ethereum log.
I will remove the duplicate.